CHANGELOG.md

```diff
@@ -29,150 +29,138 @@ Types of changes (Stanzas):
 "Client Breaking" for breaking CLI commands and REST routes used by end-users.
 "API Breaking" for breaking exported APIs used by developers building on SDK.
 "State Machine Breaking" for any changes that result in a different AppState given same genesisState and txList.
-
 Ref: https://keepachangelog.com/en/1.0.0/
 -->
 
 # Changelog
 
 ## [Unreleased]
 
-### Client Breaking
-* (modules) [\#7243](https://github.com/cosmos/cosmos-sdk/pull/7243) Rename `RegisterCodec` to `RegisterLegacyAminoCodec` and `codec.New()` is now renamed to `codec.NewLegacyAmino()`
-* (cli) [\#6651](https://github.com/cosmos/cosmos-sdk/pull/6651) The `gentx` command has been improved. No longer are `--from` and `--name` flags required. Instead, a single argument, `name`, is required which refers to the key pair in the Keyring. In addition, an optional
+### Client Breaking Changes
```

Comment:
Would it make sense to add as a first sentence that the biggest breaking change is actually the introduction of Protobuf?

something like:

"This release introduces Protobuf (with link) as the new default encoding format. Please refer to ADR-019, ADR-020 and ADR-021 (with links) to learn how Protobuf is used in the SDK."

Reply:
I was thinking this high level information would be better communicated in a RELEASE_NOTES.md ? 

Afaik there isn't any breaking client breaking changes w/r/t protobuf migration, as legacy endpoints are all preserved. Are there any way in which CLI endpoints have changed systematically due to protobuf migration?

The state machine breaking changes for each module is documented already. I can add a more general top level one that points to ADR019

I do think we should definitely add a changelog entry to "Features" or "Improvements" to illustrate the new gRPC & gRPC Gateway endpoints. I can add separate sections for Tx & Query paths that correspond to ADR020 & ADR021.

Reply:
> I was thinking this high level information would be better communicated in a RELEASE_NOTES.md ?

I believe some people will read the CHANGELOG without reading the RELEASE_NOTES, and vice versa. We could possibly add two links to reference them together.

> Afaik there isn't any breaking client breaking changes w/r/t protobuf migration, as legacy endpoints are all preserved. Are there any way in which CLI endpoints have changed systematically due to protobuf migration?

Good point regarding legacy endpoints. All CLI commands are breaking though, since we output protobuf JSON everywhere.